rosbag2_tests/test/rosbag2_tests/test_rosbag2_play_end_to_end.cpp

```diff
@@ -120,7 +120,7 @@ TEST_F(PlayEndToEndTestFixture, play_fails_gracefully_if_needed_coverter_plugin_
     error_output, HasSubstr("Requested converter for format 'wrong_format' does not exist"));
 }
 
-#ifndef _WIN32
+/*
```

Comment:
```suggestion
/*
todo(tfoote) Disabling failing test. Ticketed to follow up at: https://github.com/ros2/rosbag2/issues/408
```

Signed-off-by: Tully Foote <tfoote@osrfoundation.org>